packages/ui-components/src/components/radio-button/usage/angular.md

```diff
@@ -0,0 +1,13 @@
+```html
```

Comment:
Are we no longer adding usage for javascript?

Reply:
indeed